packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -7630,6 +7630,24 @@ const tests = {
   ],
 };
 
+if (__EXPERIMENTAL__) {
+  tests.valid = [
+    ...tests.valid,
+    {
+      code: normalizeIndent`
+      function MyComponent({ theme }) {
+        const onStuff = experimental_useEvent(() => {
```

Comment:
Seems like this needs to use the unprefixed name

Reply:
Yeah I forgot to change it, thanks!